core/src/main/scala/org/apache/spark/sql/SnappyParser.scala

```diff
@@ -607,9 +607,12 @@ class SnappyParser(session: SnappySession)
   }
 
   protected final def inlineTable: Rule1[LogicalPlan] = rule {
-    VALUES ~ push(tokenize) ~ TOKENIZE_BEGIN ~ (expression + commaSep) ~ AS.? ~ identifier.? ~
-        ('(' ~ ws ~ (identifier + commaSep) ~ ')' ~ ws).? ~>
-        ((tokenized: Boolean, valuesExpr: Seq[Expression], alias: Any, identifiers: Any) => {
+    VALUES ~ push(tokenize) ~ push(canTokenize) ~ DISABLE_TOKENIZE ~
```

Comment:
Instead of DISABLE_TOKENIZE use TOKENIZE_END. It will be consistent with rest of the places where we disable tokenization selectively and temporarily.